README.md

```diff
@@ -1,5 +1,9 @@
 # HR-TIL
 
+![til](https://raw.githubusercontent.com/michaeldimmitt/hr-til/readme-oauth-instructions/app/assets/images/banner.png)
+
+<!--![til](https://raw.githubusercontent.com/hashrocket/hr-til/master/app/assets/images/banner.png)-->
```

Comment:
@jwworth, before you commit do you want me to remove line 3 and 4 and uncomment line 5?